src/core/core.ts

```diff
@@ -46,15 +46,20 @@ const executeWithContext = function (
   resources: MilkResource[],
   context: Map<string, any>
 ): Promise<any> {
+  const logPath = (resource: MilkResource) => {
```

Comment:
Could this function be declared at the root of the file?

Reply:
like outside of `executeWithContext`? I guess it could be private. tbh I don't love the implementation yet, but I really wanted some better file logging to modify my requests directory, so I think this will change over time. This is especially true if I refactor to use a logging lib like `winston`